packages/dcos-history/extra/history/server.py

```diff
@@ -90,4 +92,10 @@ def start():
 
     state_buffer = BufferCollection(os.environ['HISTORY_BUFFER_DIR'])
     BufferUpdater(state_buffer, headers_cb).run()
-    app.run(host='0.0.0.0', port=int(os.environ.get('PORT', '15055')))
+
+
+def start():
```

Comment:
For easy local development / testing it's really nice to have a simple single command, this makes it so they all run through the same single codepath. With this for development / testing can still just run `dcos-history` as a command rather than needing to enter a python shell and manually start the server.